hbase_master/_gen.rb

```diff
@@ -17,74 +17,72 @@
 require 'csv'
 require 'erb'
 
-METRICS_FILENAME='_metrics.yml'
-OUT_CONFIG_FILENAME='_conf.yaml.example'
-OUT_METADATA_FILENAME='_metadata.csv'
-ORIENTATION = "0"
+METRICS_FILENAME = '_metrics.yml'.freeze
```

Comment:
I know this is unrelated, and nobodies fault really, but we should've probably not merged this ruby `_gen` stuff, this being a python project.